doc/user_guide/model/model_components.rst

```diff
@@ -46,6 +46,34 @@ However, this doesn't mean that you have to limit yourself to this meagre
 list of functions. As discussed below, it is very easy to turn a
 mathematical, fixed-pattern or Python function into a component.
 
+.. _compute-model-statistics:
+
+Compute model statistics
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+HyperSpy provides a method to compute and print summary statistics of all parameters
```

Comment:
The function is mainly relevant to do the calculation over the `navigation` space to get statistics for a spectral map, so that should be clarified. Statistics for a single value do not make much sense :-)